.github/workflows/deploy-test-pypi.yml

```diff
@@ -0,0 +1,18 @@
+name: Publish to TestPyPI
+
+on:
+  push:
+    branches:
+      - 'release/*.*.*'
```

Comment:
`publish-packages.yml` is not tested in this PR because it does not match this branch name. You could remove this condition temporarily to test it, or remove it permanently. GitHub Actions provides many runners, so the additional testing time is not an issue.

Reply:
This script matches the testing workflows on signac-dashboard, which were copied from signac. I think I trust this workflow from those two repos, so I was going to leave it “untested” until 0.23.0 release time. I don’t think we want to publish test packages on every branch/commit.